### ISSUE TITLE
Please add re-connect support

### ISSUE THREAD

**Issue by hedrickbt** [fastogt/fastoredis#99]
Currently have an environment with a load balancer to master/slaves.  Right now fastoredis isn't very happy when we fail a master and the slave gets promoted as it would require fastoredis to detect the connection issue and try to reconnect.

**Comment by topilski** [comment]
Hello @hedrickbt , thank you for the issue, do you mean auto-reconnect feature?

**Comment by topilski** [comment]
Ping

**Comment by hedrickbt** [comment]
Yes, please.  I have a master/slave environment load balanced by F5 and when we are performing maintenance Fastoredis does not seem to handle that without locking up and having to Lille the process and restart.

**Comment by topilski** [comment]
First of all you should use PRO version, may be as well create cluster
